docs/build-with-mingw.md

```diff
@@ -0,0 +1,107 @@
+## Building LibSass with MingGW (makefiles)
+
+First grab the latest [MinGW for windows] [1] installer. Once it is installed, you can click on continue or open the Installation Manager via `bin\mingw-get.exe`.
+
+You need to have the following components installed:
+![](https://cloud.githubusercontent.com/assets/282293/5525466/947bf396-89e6-11e4-841d-4aa916f14de1.png)
```

Comment:
Feel free to edit as necessary: "A list of of packages to install: mingw-developer-toolkit, mingw32-base, mingw32-gcc-ada, mingw32-gcc-fortran, mingw32-gcc-g++, mingw32-gcc-objc, msys-base."